framework/include/indicators/ElementIndicator.h

```diff
@@ -62,9 +62,15 @@ class ElementIndicator : public Indicator,
   /// Time derivative of u
   const VariableValue & _u_dot;
 
+  /// Second time derivative of u
```

Comment:
A general discussion: I am not sure if higher order time derivatives are common parameters for element indicators (and some other classes we modified) in the sense that they should be put in the base class.
If requesting these time derivatives is not such big a burden then we don't need worry.